Source/WebCore/platform/graphics/PlatformDisplay.cpp

```diff
@@ -221,6 +233,10 @@ PlatformDisplay::~PlatformDisplay()
 #if PLATFORM(GTK)
     if (m_sharedDisplay)
         g_signal_handlers_disconnect_by_data(m_sharedDisplay.get(), this);
+#endif
+#if USE(EGL) && USE(GBM)
+    if (m_gbm.device.has_value() && m_gbm.device.value())
+        gbm_device_destroy(m_gbm.device.value());
```

Comment:
This should clear `m_gbm.device` as well, for the sake of completeness.

```suggestion
    if (m_gbm.device.has_value() && m_gbm.device.value()) {
        gbm_device_destroy(m_gbm.device.value());
        m_gbm.device.reset();
    }
```

Reply:
We don't usually do that in destructors.

Reply:
Fair enough 😄